napari/_vispy/vendored/filters/clipping_planes.py

```diff
@@ -0,0 +1,95 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) Vispy Development Team. All Rights Reserved.
+# Distributed under the (new) BSD License. See LICENSE.txt for more info.
+
+from functools import lru_cache
+
+import numpy as np
+
+from vispy.visuals.shaders import Function, FunctionChain, Varying
+from .base_filter import Filter
+
+
+class PlanesClipper(Filter):
```

Comment:
I don't love this name, was there a reason for calling it this as opposed to 'ClippingPlanes'?